keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -56,6 +52,9 @@ class SqlVerificationRepository(
   specMigrators
 ), VerificationRepository {
 
+  private val useLockingRead : Boolean
+    get() = environment.getProperty("keel.verifications.db.lock.reads.enabled", Boolean::class.java, true)
```

Comment:
Shouldn't we inject this using `@Value` rather than using the low-level Spring API?

Reply:
I can't use a `@Value` here because (AFAIK) that doesn't let us change the value of a property at runtime via fast properties. Using the `getProperty` method is the only mechanism I know of that will enable the code to respect a fast property change.